src/main/java/com/codegym/shopyy/controller/ProductDetailControllerIpa.java

```diff
@@ -0,0 +1,48 @@
+package com.codegym.shopyy.controller;
```

Comment:
delete this controller, we just use ProductController for all APIs of Product, ProductDetail